server/config/config_test.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+import (
+	"testing"
+)
+
+func TestConfig_Load(t *testing.T) {
+	t.Setenv("JPRQ_DOMAIN", "jprq.live")
+	t.Setenv("JPRQ_TLS_KEY", "key.pem")
+	t.Setenv("JPRQ_TLS_CERT", "cert.pem")
+
+	config := &Config{}
+	err := config.Load()
+	if err != nil {
+		t.Logf("Error while loading the config: %v", err.Error())
+		t.Fail()
+	}
+}
+
+func TestConfig_LoadEmptyEnv(t *testing.T) {
+	cases := []struct {
```

Comment:
Number of test cases will go up to 2^n (where n is number of env vars)
Let's write N test cases instead.